docs/griptape-cloud/knowledge-bases/accessing-data.md

```diff
@@ -0,0 +1,33 @@
+# Accessing Data in a Knowledge Base
+
+You can `Search` or `Query` the Knowledge Base for information contained in your Data Sources. `Search` will return a natural language response while `Query` will return the individual entries. Use whichever one best fits your use case.
+
+## From the UI
```

Comment:
I could understand if we're worried about things getting out of date, but I'd love to see some screenshots for anything that references actions you can take in the UI.